x-pack/plugins/cloud_security_posture/public/pages/configurations/findings_flyout/overview_tab.tsx

```diff
@@ -161,11 +173,17 @@ const getEvidenceList = ({ result }: CspFinding) =>
   ].filter(truthy);
 
 export const OverviewTab = ({ data }: { data: CspFinding }) => {
-  const {
-    services: { discover },
-  } = useKibana();
+  const { discover, application } = useKibana().services;
   const latestFindingsDataView = useLatestFindingsDataView(LATEST_FINDINGS_INDEX_PATTERN);
 
+  const ruleFlyoutLink = application?.getUrlForApp('security', {
+    path: generatePath(benchmarksNavigation.rules.path, {
+      benchmarkVersion: data.rule.benchmark.version.split('v')[1], // removing the v from the version
```

Comment:
> Though quite small, it's still a piece of logic, I'd move it out and add basic unit tests for it. Plus we already use it in two places.

After sharing the logic between the tabs there is only one occurrence for this logic, at least for the url needs. Regarding testing, i don't really see what we need to test here since its just a basic javascript `split` method. the only unexpected outcome would be from a defective data, which we can't really account for. if the arrived data is defective what can we do about it?

> btw why do we have this inconsistency between the path and the data, wouldn't it be easier to just have v.. everywhere?

good question, i didn't pay much attention to it and just created the url as it was expected to work. i will take a look on how complex will this change is and try to implement it. if i'll manage to change the route to include the `v` we could also remove the above logic completely which is good

Reply:
> i don't really see what we need to test here since its just a basic javascript `split` method

agree that it's a super small piece of logic, my personal preference to still have a unit test over it for two reasons:
- I'm quite a visual person and tests often serve me as documentation. Kind of. Eg. it's much easier to understand what we are doing here when I see the test for `prepareBenchmarkVersionForUrlPath('v1.1.1')` returning `1.1.1` than just looking at
`data.rule.benchmark.version.split('v')[1]` and having to think what can be in `data.rule.benchmark.version` and why do we want to split by `v` and take only the second part
- having a test protect to some extent for future additions to the logic. eg. in future we would like to say not only remove `v` from the version, but also remove `-beta` in the end (i'm coming up with things here). The more logic we add, the more chances by mistake to remove the original piece of logic. The unit test would make sure we don't have regressions when touching this piece of code

It's a nit ofc as the logic here is super straightforward, just sharing how I reason about it

> we can open a tech debt if we think its worth investing the time into that

I guess there might be quite a lot of side-effects in terms of backwards compatibility (eg. alerts created with tags without `v` or smth). I leave the need to the tech debt ticket to you, if you didn't see big problems with the need to remove `v` in some places while working on it, maybe it's not a big deal then